README.CentOS7.md

```diff
@@ -103,6 +103,48 @@ Customize and Verify your settings:
  * DocumentRoot should point to `/var/www/loris/htdocs`
  * The `smarty/templates_c/` directory must be writable by Apache (e.g. by running: `sudo chgrp -R httpd /var/www/loris/smarty/templates_c` and `sudo chmod 775 /var/www/loris/smarty/templates_c`).
 
+Create the Apache configuration `/etc/httpd/conf.d/loris.conf` for your LORIS environment. Adjust the following parameters to according to your configuration
+
+- %LORISROOT%    i.e. /var/www/loris
+- %PROJECTNAME%  i.e loris
+- %LOGDIRECTORY%  .i.e /var/log/httpd
+```
+<VirtualHost *:80>  # change from 80 to 443 if you enable SSL
```

Comment:
1. Why is this duplicating the docs/config/apache2-site file instead of simply referencing it? It's not going to get maintained here
2. the paths shouldn't include "/var/www", that's part of the %LORISROOT% variable.

Reply:
@driusan can you review the latest changes. I referenced the loris/docs/config/apache2-site and removed the VirtualHost example.